app/assets/stylesheets/helpers/_variables.scss

```diff
@@ -18,7 +18,7 @@ $text-body: #555555;
 $element-toned-down: #bbbbbb;
 $text-toned-down: #777777;
 $text-light: #ffffff;
-$facebook-blue: #4267b2;
+$facebook-blue: #3C5A99;
```

Comment:
What's the reason for changing this? #4267b2 is what we currently have on the style guide in zeplin.